### ISSUE TITLE
Implement Age 4 cards for Artifacts Expansion

### ISSUE THREAD

**Issue by micahstairs** [micahstairs/bga-innovation#104]
This issue will track the implementation of all Age 4 cards for the Artifacts expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

  - [x] **145 - Petition of Right**
  - [x] **146 - Delft Pocket Telescope**
  - [x] **147 - East India Company Charter**
  - [x] **148 - Tortugas Galleon**
  - [x] **149 - Molasses Reef Caravel**
  - [x] **150 - Hunt-Lenox Globe**
  - [x] **151 - Moses**
  - [x] **152 - Mona Lisa**
  - [x] **153 - Cross of Coronado**
  - [x] **154 - Abell Gallery Harpsichord**
